docs/content.zh/docs/concepts/glossary.md

```diff
@@ -78,6 +78,13 @@ A Flink Job is the runtime representation of a [logical graph](#logical-graph)
 
 见 [Logical Graph](#logical-graph)。
 
+#### JobResultStore
```

Comment:
The `JobResultStore` should be located in front of the `Logical Graph` entry because of the lexicographical order.